<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBoxNotCalcature">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>651</width>
     <height>451</height>
    </rect>
   </property>
   <property name="title">
    <string>Not Hesaplama Otomasyonu</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>168</x>
      <y>40</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Öğrenci Adı:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>168</x>
      <y>80</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Ders Adı:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>168</x>
      <y>120</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Vize Notu</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>168</x>
      <y>160</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Final Notu:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditStudentName">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditLessionName">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditVizeNote">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFinalNote">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserResult">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>611</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>381</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnCalcature">
       <property name="text">
        <string>Hesapla</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClear">
       <property name="text">
        <string>Temizle</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>520</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Çıkış</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnExit</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>678</x>
     <y>551</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>574</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
